pages/studentPortal/quizcomponent/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react";
+
+type SkillSelectionProps = {
+  selections: string[];
+};
+
+const SkillSelection: React.FC<SkillSelectionProps> = ({ selections }) => {
+  const [selected, setSelected] = useState<string[]>([]);
+
+  const handleSelection = (selection: string) => {
+    if (selected.includes(selection)) {
+      setSelected(selected.filter((s) => s !== selection));
+    } else {
+      setSelected([...selected, selection]);
+    }
+  };
+
+  return (
+    <div className="flex flex-col w-full max-w-4xl mx-auto p-4">
+      <div className="w-full mb-4 text-center">
+        <div className="font-bold text-2xl text-black-600"></div>
+      </div>
+      {selections.map((selection, index) => (
+        <div
+          key={index}
+          className={`flex   items-start justify-start w-full px-4 py-2 my-2  cursor-pointer ${
+            selected.includes(selection)
+              ? "bg-violet-300 text-black-500 border-2 border-black-500 rounded-[14px]"
```

Comment:
the border-2 border-black-500 rounded-[14px] could be move outside of the ternary. This would make it easier to understand how the the ternary condition affects the style.

Also, I'm not big on using customized hard-coded values in tailwind styles, unless necessary. Do the pre-defined rounded tailwind classes not work for you? like rounded-lg?

Reply:
Yeah they do work, I'll use the defaults.